config.py

```diff
@@ -84,13 +84,26 @@ class BaseConfig:
     # Services
     # --------
 
+    #: URL for mail server.
+    MAIL_SERVER = _env("MAIL_SERVER")
+    #: Port for mail server.
+    MAIL_PORT = _env("MAIL_PORT")
```

Comment:
Where are these variables used? Are they picked up automatically by flask-mailer?

Reply:
@akprasad ?

Reply:
Added a comment above that these are for `Flask-Mail`, maybe the GH preview cuts it off